apps/shipments/serializers.py

```diff
@@ -227,7 +227,7 @@ def validate_device_id(self, device_id):
         return device_id
 
     def validate_delivery_act(self, delivery_act):
-        if delivery_act <= datetime.now():
+        if delivery_act <= datetime.utcnow():
```

Comment:
I am fairly certain that this comparison won't function propertly, as delivery_act is a timezone-aware datetime and datetime.utcnow() is timezone-naive: https://stackoverflow.com/questions/2331592/why-does-datetime-datetime-utcnow-not-contain-timezone-information

I think I like this solution: `datetime.datetime.now(datetime.timezone.utc)`